exchanges/coinbene/coinbene.go

```diff
@@ -5,6 +5,7 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	currency2 "github.com/thrasher-corp/gocryptotrader/currency"
```

Comment:
There are no package collisions so currency2 can be dropped and called with just currency